### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by prinshul** [StanfordMIMI/DDM2#4]
Hi,
Thank you for sharing your code.

Can you please share the pre-trained models?

**Comment by tiangexiang** [comment]
Thank you for your interest in our work! Unfortunately, we don't plan to release the trained model, since we have refactored the code quite a lot and the trained model cannot be loaded directly in the current repo due to inconsistent names/structures. Note that our method is dataset-specific, such that the trained model on one dataset can not be used to denoise any other datasets.
However, we do provide the denoised data for all the four datasets we presented in the paper: https://figshare.com/s/6275f40c32f67e3b6083

**Comment by prinshul** [comment]
Sure. I will train the model from scratch. Waiting for the metric calculation script on test data for Hardi dataset. Thanks.
